services/credentials.go

```diff
@@ -102,15 +103,15 @@ func GetQuotaJSON(ot credentials.OperatorType) (json.RawMessage, error) {
 
 // Creates a new credential for a node. If a valid credential already exists, it will be returned instead.
 // This method will retry if creating a credential fails.
-func (s *Service) CreateCredentialWithRetry(msg []byte, sig []byte, ot credentials.OperatorType) (*models.AuthenticatedCredential, error) {
+func (s *Service) CreateCredentialWithRetry(msg []byte, sig []byte, dataHash common.Hash, address common.Address, ot credentials.OperatorType) (*models.AuthenticatedCredential, error) {
 	var cred *models.AuthenticatedCredential
 	var err error
 
 	var try int
 	s.m.Counter("create_credential_with_retry").Inc()
 	for try = range dbTryDelayMs {
 		// Try to create the credential.
-		if cred, err = s.CreateCredential(msg, sig, ot); err == nil {
+		if cred, err = s.CreateCredential(msg, sig, dataHash, address, ot); err == nil {
```

Comment:
All we're doing is passing these dataHash and address objects down into `CreateCredential`.